### ISSUE TITLE
Rebound effecting `dx`, `dy` when combining `restrict` and `inertia` with `endOnly: false`

### ISSUE THREAD

**Issue by nayfin** [taye/interact.js#765]
### Expected behavior

When restricting a drag item with `inertia:{resistance: 1}` to its parent with `{endOnly: false}`, flinging the drag item at the side of the parent until it stops, then quickly drag the item again: the item should appear below the cursor and drag normally. 
### Actual behavior

The item jumps away from the wall it was flung at, and the first first `dx/dy` values emitted upon redragging element are very high and seem to be inversely proportional to the distance flung.

This appears to be due to some logic that brings the drag item back when restricting with option of `{endOnly: true}` because if, after flinging the drag element, you wait a few seconds everything behaves as expected.

Please see repro [here](http://jsfiddle.net/natjkern/y4sxc9h3/11/). I set it up so that it will only log events with large `dx`/`dy` values, in order to highlight the issue.

Also, this appears to be related to a recent change as I could not reproduce behavior in `interact.js 1.29`

Please let me know if I can help reproducing issue or debugging. Thank you for this great drag library!

#### EDIT:
I just found the `zeroResumeDelta` option for inertia, and thought it would fix the issue, but it did not. I updated the fiddle to reflect, I've tried `true` and `false` in case I read the docs wrong. Additionally, it looks like `zeroResumeDelta` needs to be added to the type definitions, as I get type error when trying to use it in a TypeScript context.

### System configuration
**interact.js version**: 1.6.2
**Browser name and version**: Chrome @latest
**Operating System**: Windows 10

**Comment by nayfin** [comment]
I spent some time digging through source and it's history. It looks `zeroResumeDelta` was removed, [here](https://github.com/taye/interact.js/commit/f70533614e878b25b7b098ecac2d774775da4238), years ago, and replaced with a new `inertiaresume` event, which also seems to have since been removed.

I did find the 'dragresume' event, but I can't figure out how to keep it from propagating its deltas to the 'dragmove' event in order to prevent this behavior. I have tried the following:
```typescript
interact('#some-element').draggable(this.dragConfig).on('dragresume', (event: DragEvent) => { 
  event.delta = {x: 0, y: 0}; 
  event.preventDefault();
  event.stopImmediatePropagation(); 
  event.stopPropagation(); 
});
```
This hasn't helped.

I feel like I must be missing something. @taye if you can help me to figure out what I am missing, I can make a PR to update docs, so that you don't have to explain it again.

**Comment by nayfin** [comment]
@taye thank you for the fast fix and release of 1.6.3!

**Comment by taye** [comment]
You're welcome!
